pkg/util/kube/utils.go

```diff
@@ -126,6 +126,10 @@ func GetVolumeDirectory(pod *corev1api.Pod, volumeName string, pvcLister corev1l
 		return "", errors.WithStack(err)
 	}
 
+   	if os.Getenv("ONLY_CSI") != "" {
```

Comment:
Where is this defined? Can we key off some other information to get at this rather than setting an environment variable?

Reply:
Hello,
this variable should be defined in the docker-compose.yml using Velero's image. If not define, then de default behavior will be conserved.